updater/src/Updater/SyncLabels/IssueLabel.purs

```diff
@@ -35,27 +35,27 @@ issueLabelCodec =
 labels :: Array IssueLabel
 labels =
   [ { name: "breaking change"
-    , description: "Change will require a major version bump"
+    , description: "A change that requires a major version bump"
     , color: "e99695"
     }
   , { name: "bug"
-    , description: "Something isn't working"
+    , description: "A legitimate bug in the library that ought to be fixed"
     , color: "d73a4a"
     }
   , { name: "document me"
     , description: "Improvements or additions to documentation"
     , color: "0075ca"
     }
   , { name: "enhancement"
-    , description: "New feature or request"
+    , description: "An addition to the library"
     , color: "a6e1ea"
     }
   , { name: "good first issue"
     , description: "Good for newcomers"
     , color: "7057ff"
     }
   , { name: "help wanted"
-    , description: "Extra attention is needed"
+    , description: "Maintainers would like a PR that solves this issue"
```

Comment:
I'd like to propose a slight change to the wording here:

```diff
- Maintainers would like a PR that solves this issue
+ Maintainers would like assistance with solving this issue
```

I think by specifically pointing out that we'd like a PR we could be discouraging potential contributors (especially ones who may not be as familiar with PRs/open source as a whole).

An example of this that comes to mind is that someone could comment on a "help wanted" issue with reproduction steps or links to some useful resources. These are things that would progress the issue towards resolution, perhaps opening the door for them or someone else to open a PR that resolves the issue.

Reply:
I like this wording and I agree with the reasoning.